tests/test_sender.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3.5m
+#
+#   Copyright (C) 2017, Collabora Ltd.
+#
+#   This Source Code Form is subject to the terms of the Mozilla Public
+#   License, v. 2.0. If a copy of the MPL was not distributed with this file,
+#   You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import sys
+import unittest
+import numpy as np
```

Comment:
I don't think we need to use numpy, does the vsi python bindings use it? I understand the spec has different types, but from our conversations with jlr they are ok with using native types in languages such as python, as long as the correct types are used in C/C++ code (which is something the bindings are responsible for).